core/lib/zksync_core/src/sync_layer/tests.rs

```diff
@@ -181,9 +181,11 @@ async fn external_io_basics() {
 
     let tx_receipt = storage
         .transactions_web3_dal()
-        .get_transaction_receipt(tx_hash)
```

Comment:
Nit: You could preserve the `get_transaction_receipt` method in DAL, which would delegate to `get_transaction_receipts` with a single-item slice argument. Could simplify call sites a bit in places like this.